src/style.css

```diff
@@ -25,6 +25,188 @@ ul {
 }
 
 a {
-  text-decoration: none;
+  color: black !important;
+}
+
+.site-header {
+  background-color: orange;
+  color: black;
+}
+
+.hero1 {
+  background: url('../src/assets/images/beef.png') no-repeat left bottom;
+}
+
+.hero2 {
+  background: url('../src/assets/images/miscellaneous.png') no-repeat right bottom;
+}
+
+.showbtn {
+  cursor: pointer;
+  background: salmon;
+  border: 2px solid white;
+  color: white;
+  font-size: 16px;
+  padding: 5px;
+}
+
+.actions button {
+  background: none;
+  cursor: pointer;
+  border: none;
   color: black;
+  font-size: 14px;
+  padding: 5px;
+}
+
+.actions button:hover {
+  border-bottom: 2px solid black;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.main {
+  width: 90%;
+  gap: 20px;
+  overflow-x: scroll;
+}
+
+.mainTitle {
+  font-size: 50px;
+}
+
+.singleContainer {
+  background-color: antiquewhite;
+  padding: 20px;
+}
+
+.singleContainer img {
+  display: block;
+}
+
+.description {
+  max-height: 52px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-bottom: 25px;
+}
+
+.mealImg {
+  cursor: pointer;
+  max-width: 90%;
+  border-radius: 15px;
+}
+
+.mealContent {
+  width: 40%;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+
+.mealName {
+  max-height: 20px;
+  overflow: scroll;
```

Comment:
- [ ] due to this `overflow: scroll` styling all the mealNames are not visible, I would advise you to change this to `auto`